node/cli/Cargo.toml

```diff
@@ -77,12 +77,14 @@ default = ["std", "cli"]
 runtime-benchmarks = [
   "polkadot-cli/runtime-benchmarks",
   "frequency-runtime/runtime-benchmarks",
+  "sc-service?/runtime-benchmarks",
```

Comment:
Q: why this has `?` ?

Reply:
I noticed that it was configured that way in substrate. I have made a note of it to see if we can remove it on my follow up PR.